File: cw.py
#!/usr/bin/env python
import time, random; D=64; _=[random.randint(0,1) for i in range(0,D*D)]
while time.sleep(0.2) or True:
    print chr(0xa).join(map(''.join, zip(*[iter([chr(32+3*j) for j in _])]*D)))
    p=lambda p:_[(p[0]%D)+D*(p[1]%D)];n=lambda x,y:reduce(lambda x,y:x+y,map(p,
    [(x+i%3-1,y+i/3-1)for i in range(0,9) if i!=4]));_=[int(((p((j%D,j/D))|n(j%
    D,j/D))==3)or(~p((j%D,j/D))and n(j%D,j/D)==3))for j in range(0,D*D)]

File: cw.rb
D=32;d=(0...D*D);b=d.map{rand 2};p=->y,x{b[x%D+y%D*D]}
n=->y,x{[*0..3,*5..8].map{|i|p[y+i/3-1,x+i%3-1]}.reduce :+}
loop{sleep 1;puts"\033[2J";b.map{|c|(32+3*c).chr}.each_slice(D){|p|p p.join}
b=d.map{|j|m=j/D,j%D;p[*m]|n[*m]==3||~p[*m]&&n[*m]==3?1:0}}
